version: "3.7"
services:
  app:
    image: "172.31.249.114:5005/pds/proto:latest"
    ports:
      - "5000:5000"
    networks:
      - pypoc
    depends_on:
      - postgres

  postgres:
    image: "postgres:11"
    ports:
      - "5432:5432"
    networks:
      - pypoc
    environment:
      POSTGRES_USER: pds
      POSTGRES_PASSWORD: pds
      POSTGRES_DB: pds
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U pds"]
      interval: 10s
      timeout: 5s
      retries: 5
    volumes:
      - pg_data:/var/lib/postgresql/data

volumes:
  pg_data:

networks:
  pypoc: